test-dts/mount.d-test.ts

```diff
@@ -211,6 +212,48 @@ class ClassComponent extends Vue {
 expectError(mount(ClassComponent, {}).vm.changeMessage())
 mount(ClassComponent, {}).vm.changeMessage('')
 
+// region other class component implement
+class OtherClassComponent<Props extends {} = {}> {
+  private static __vccValue?: ComponentOptions
+  static get __vccOpts(): ComponentOptions {
+    if (this.__vccValue) return this.__vccValue
+    const CompConstructor = this
+    return this.__vccValue = {
+      name: OtherClassComponent.name,
+      props: this.defaultProps,
+      setup(props, ctx) {
+        const instance = new CompConstructor()
+        instance.props = props
+        instance.context = ctx
+        ctx.expose(instance)
+        return instance.render.bind(instance)
+      }
+    }
+  }
```

Comment:
OOC: Is this something that developers really write? Why not extend the base `Vue` class?

Reply:
where is the Vue class? i use it in my library [vue3-oop ](https://github.com/agileago/vue3-oop)

Reply:
`vue-class-component` exports a `Vue` class that components should extend (if I remember correctly).